src/library/VUMeter.js

```diff
@@ -2,20 +2,21 @@ import FIFO from './FIFO.js';
 
 /**
  * This VUMeter class implements a volume level meter UI. To initialize it,
- * you need to provide the ID of the canvas element where the waveform will 
- * be rendered, and the minimum decibel value for the VUMeter. The minimum 
- * decibel value defines the range of the VUMeter that will be considered 
- * silent. When the waveform is created, it requires an AnalyserNode for 
- * real-time audio input data. This class only requires a canvas element, 
- * and there is no additional dependency.
+ * you need to provide the ID of the canvas element where the VUMeter will 
+ * be rendered, the minimum decibel value for the VUMeter, analyserNode of
+ * audio context, fftSize of the analyserNode, and fifoSize for control the
+ * VMMeter refresh speed. The minimum decibel value defines the range of 
+ * the VUMeter that will be considered silent. When the VUMeter is created, 
+ * it requires an AnalyserNode for real-time audio input data. This class 
+ * only requires a canvas element, and there is no additional dependency.
```

Comment:
It does have dependency. (e.g. `import` at the top)

Please explain why this dependency is needed and how FIFO is used in the class.